velox/exec/HashTable.h

```diff
@@ -125,6 +125,7 @@ class BaseHashTable {
 
   /// The name of the runtime stats collected and reported by operators that use
   /// the HashTable (HashBuild, HashAggregation).
+  static inline const std::string kBuildWallNanos{"hashtable.buildWallNanos"};
```

Comment:
let's clarify that this stat is only reported by the HashBuild operator.

Reply:
Fixed